src/main/java/com/redhat/documentation/asciidoc/cli/ExtractionRunner.java

```diff
@@ -77,22 +74,51 @@ public Integer call() {
             writeAssemblies(config);
         }
 
+        int errors = 0;
+        for (Issue issue : this.issues) {
+            if(issue.isError()) {
+                errors++;
+            }
+        }
```

Comment:
You could probably reduce this using a stream and a filter, probably easier to read.